serving-core/src/main/scala/com/stratio/sparkta/serving/core/helpers/JarsHelper.scala

```diff
@@ -42,24 +42,27 @@ object JarsHelper extends SLF4JLogging {
                      notContains: Option[String] = None,
                      excludeFolder: Option[Seq[String]] = None,
                      doAddToClassPath: Boolean = true): Seq[File] = {
-    if (!path.isDirectory ||
-      (path.isDirectory && excludeFolder.forall(folder => folder.forall(exFolder => path.getName != exFolder)))) {
+    if (path.exists && (!path.isDirectory ||
+      (path.isDirectory && excludeFolder.forall(folder => !folder.contains(path.getName))))) {
```

Comment:
I think that you should extract this big conditional to a function in order to improve the readability